src/components/Select/styled/label.js

```diff
@@ -3,13 +3,19 @@ import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
 import { FONT_SIZE_TEXT_MEDIUM } from '../../../styles/fontSizes';
 import { MARGIN_SMALL, MARGIN_XX_SMALL } from '../../../styles/margins';
 
+const LABEL_ALIGNMENT_MAPPING = {
+    center: 'center',
+    left: 'flex-start',
+    right: 'flex-end',
+};
+
 const StyledLabel = attachThemeAttrs(styled.label)`
     color: ${props => props.palette.text.label};
     font-size: ${FONT_SIZE_TEXT_MEDIUM};
     line-height: 1.5;
     margin-right: ${MARGIN_SMALL};
     margin-bottom: ${MARGIN_XX_SMALL};
-    align-self: center;
+    align-self: ${props => LABEL_ALIGNMENT_MAPPING[props.labelAlignment]};
```

Comment:
when a user passes a wrong labelAlignment value that does not exist in the map we should fallback to 'center' like:
``` align-self: ${props => LABEL_ALIGNMENT_MAPPING[props.labelAlignment] || 'center'};```

Reply:
done